README.md

```diff
@@ -5,13 +5,14 @@ WSO2 Balana Implementation
 
 |  Branch | Build Status |
 | :------------ |:-------------
-| master      | [![Build Status](https://wso2.org/jenkins/job/wso2-balana/badge/icon)](https://wso2.org/jenkins/job/wso2-balana) |
-
+| master      | [![Build Status](https://wso2.org/jenkins/buildStatus/icon?job=forked-dependencies/wso2-balana)](https://wso2.org/jenkins/job/forked-dependencies/wso2-balana) |
```

Comment:
According to [1] both of the build status badges are correct. One is protected and other one is not protected.

[1] https://wso2.org/jenkins/job/forked-dependencies/job/wso2-balana/badge/